crates/pallet-feeds/src/mock.rs

```diff
@@ -56,16 +59,33 @@ parameter_types! {
     pub const MaxFeeds: u32 = 1;
 }
 
+#[derive(Debug, Clone, Encode, Decode, TypeInfo, Eq, PartialEq)]
```

Comment:
I'm still not entirely happy that this has any content at all and the type in the pallet is not longer a copy type. This is really not how actual feed processors are working right now and will work, so changing pallet in such a way just to write these unconventional mocks looks really weird to me.

Reply:
fair enough!

Reply:
> and will work, 

I disagree. Feed processors will need to hold some information. For example Contract address. This was as part of the initial discussion IIRC.

True I'm using it unconventionally but I would still like to keep it.

Reply:
Contract address will be something like `[u8; 32]`, which is a copy type too. Either way, we can leave it as is for now.

Reply:
Actually, true. I wasn;t derving the Copy trait when trying with [u8; 32] earlier :(
We can remove it